pom.xml

```diff
@@ -0,0 +1,95 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>uk.gov.companieshouse</groupId>
+        <artifactId>companies-house-parent</artifactId>
+        <version>1.3.0</version>
+        <relativePath/>
+    </parent>
+
+    <artifactId>item-group-workflow-api</artifactId>
+    <version>unversioned</version>
+    <name>item-group-workflow-api</name>
+    <description>Rest API to handle item group workflow requests.</description>
+    <packaging>jar</packaging>
+
+    <properties>
+        <java.version>11</java.version>
+        <spring-boot-dependencies.version>2.7.8</spring-boot-dependencies.version>
+        <maven.compiler.source>11</maven.compiler.source>
+        <maven.compiler.target>11</maven.compiler.target>
```

Comment:
Are these still needed with the spring-boot-maven-plugin?

Reply:
will test and let you know